byway/src/main/webapp/routepage.html

```diff
@@ -6,16 +6,24 @@
     <link rel="stylesheet" href="/bootstrap/bootstrap.min.css">
```

Comment:
Choose to use bootstrap as either CDN or move the dowloaded version (preferred by Landon) into this directory, not both.

Reply:
Fixed, used the downloaded version